lib/Wikia/src/Service/User/Preferences/PreferenceService.php

```diff
@@ -43,6 +43,9 @@ public function __construct(
 		$this->defaultPreferences = $defaultPrefs;
 		$this->forceSavePrefs = $forceSavePrefs;
 		$this->preferences = [];
+
+		// 🐖: anonymous users will always have default preferences
```

Comment:
Let's avoid using UTF characters in code comments. Ticket ID will be way more useful here :)